### DISCUSSION TITLE
By U.S. law, exit signs must be one of what two colors?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#60] [Galaxy Trivia] [answered]
**Hint:** RGB

**Comment by Tynab** [comment, marked as answer]
By U.S. law, exit signs must be one of two colors: red or green. Red is the most common color for exit signs, but green can also be used in certain situations, such as in theaters and other venues where the audience is accustomed to seeing green exit signs. The use of these two colors is intended to ensure that exit signs are easily recognizable and distinguishable from other signs and symbols in an emergency situation. Additionally, the text on exit signs must be in plain, legible letters and the sign must be illuminated or self-luminous to ensure visibility in low light or smoke-filled environments.
